libs/blocks/section-metadata/section-metadata.js

```diff
@@ -73,4 +79,5 @@ export default async function init(el) {
   if (metadata.layout) handleLayout(metadata.layout.text, section);
   if (metadata.masonry) handleMasonry(metadata.masonry.text, section);
   if (metadata.delay) handleDelay(metadata.delay.text, section);
+  if (metadata.anchor) handleAnchor(metadata.anchor.text, section);
```

Comment:
How does this work for long sections? Assuming I have a section with 3 long blocks and I want to link to the 3rd, wouldn't I be taken to the top f the section? Or are we suggesting that blocks that should have anchors be separated into their own sections?

Reply:
That may be so, but we need to make sure this implementation satisfies GWP's needs and doesn't cause re-authoring overhead.

Reply:
You don't think it will or you'd just like to see confirmation from GWP folks that they're happy with it?

Reply:
I think there will be use-cases where authors will need to make some changes in regards to sections (e.g. place the block into a separate section), but this downside is quite small compared to the benefits of using a solution tested in Production (Express) and agreed by the EDS team.